### PR TITLE
669 code cleanup

### LINKED ISSUE #669: Code cleanup
This task will constitute 

1. Removing unused classes and functions
2. Fixing spacing 
3. Optimizing imports and formatting to conform with the PEP standard
4. Removing print statements or converting them to logging statements

### PR DESCRIPTION
**Pull request by ecowan**
Closes #669

### CODE REVIEW

**Comment by ecowan** [comment]
This PR constitutes a clean up of nearly every corner of the DP Creator repo. In particular, I focused on removing unused code and reorganizing the remaining code to be PEP-compliant and more consistent in terms of spacing and styling. The only notable changes to functionality itself is several functions which had unused arguments. 

For the future I would recommend using a consistent linter (we had discussed a Github action, but the idea of a build failing due to PEP seemed overly harsh).

File: server/opendp_project/settings/development.py [hunk 1]
```diff
@@ -1,6 +1,3 @@
-from opendp_project.settings.base import *
```
Comment (raprasad) [inline server/opendp_project/settings/development.py]: Removing this line causes an error

**Review by raprasad — COMMENTED** [review]
Local testing looks good. (Restored an import line)

**Review by raprasad — APPROVED** [review]
approved

**Pull request merged**